src/stratigraphy/util/layer_identifier_column.py

```diff
@@ -92,6 +116,48 @@ def is_contained(self, rect: fitz.Rect) -> bool:
             and self.rect().y1 <= rect.y1
         )
 
+    def get_depth_interval(self, block: TextBlock) -> LayerDepthColumnEntry:
+        """Extract depth interval from a material description block.
+
+        For borehole profiles in the Deriaz layout, the depth interval is usually found in the text description
+        of the material. Often, these text descriptions contain a further separation into multiple sub layers.
+        These sub layers have their own depth intervals. This function extracts the overall depth interval,
+        spanning across all mentioned sub layers.
+
+        Args:
+            block (TextBlock): The block to calculate the depth interval for.
+
+        Returns:
+            LayerDepthColumnEntry: The depth interval.
+        """
+        depth_entries = []
+        for line in block.lines:
+            try:
+                new_entries = extract_layer_depth_interval_entries(line.text, line, require_start_of_string=False)
```

Comment:
Can you explain why `require_start_of_string=False` is necessary here?

Reply:
There are example where the depth value is not at the start of a textline. 

Something like: "Remblais Heterogene: 0.00 - 0.5m"

Reply:
Add a comment to the code.